pycbc/catalog/__init__.py

```diff
@@ -40,31 +40,42 @@
 _aliases['z'] = _aliases['redshift'] = 'redshift'
 _aliases['distance'] = 'luminosity_distance'
 
+def find_event_in_catalog(source='any'):
```

Comment:
I suggest using None in place of the string 'any' (same in the constructor of Merger, below).

Reply:
Ok